### DISCUSSION TITLE
No such container: so-idstools on Sensor

### DISCUSSION THREAD

**Discussion by kylehill229** [Security-Onion-Solutions/securityonion#8438] [Unsupported Versions] [answered]
Current Architecture:  Distributed with 1 Manager+Search Node and 1 Forward Sensor Node

We are currently able to query PCAP from our sensor via the SOC web UI, but are not receiving any nids alerts.  There is an so-suricata container running on the sensor (there is not on the Manager+Search).  However, the nids rule files only appear to be exist on the Manager+Search node.  Trying to perform a rule update (via salt or direct on each) succeeds on the Manager+Search, but gives the no such container: so-idstools error on the sensor.

Just looking for some guidance on if I should have deployed something differently with the sensor (or if nids tap should be fed directly to Manager+Search instead and only hids should be on the forward node?  Or if we can now adjust its config?

Thank you for suggestions / direction!!
-Kyle

**Comment by dougburks** [comment, marked as answer]
> There is an so-suricata container running on the sensor (there is not on the Manager+Search). However, the nids rule files only appear to be exist on the Manager+Search node. Trying to perform a rule update (via salt or direct on each) succeeds on the Manager+Search, but gives the no such container: so-idstools error on the sensor.

All of this sounds correct. `so-idstools` is only used on the manager to download rules from the Internet. Sensors then sync rules from the manager via salt.

> We are currently able to query PCAP from our sensor via the SOC web UI, but are not receiving any nids alerts. 

Have you checked the Troubleshooting Alerts section of our documentation?
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts

**Reply by kylehill229** [reply]
Thanks Doug!  It was the details of the Homenet for the sensor
